main.tf

```diff
@@ -38,6 +38,11 @@ data "aws_ssm_parameter" "zone_id" {
   name = "/acs/dns/zone-id"
 }
 
+// CodePipeline parameters
+data "aws_ssm_parameter" "github_token" {
```

Comment:
Maybe add a conditional here if the parameter doesn't exist (in dept accounts) so that terraform doesn't blow up

Reply:
That makes sense. Where should the data for the conditional come from? Should it be an input variable, requiring the user to specify if this is an OIT account or not? Or should that be something that ACS gives us (another ssm param)? Or should we infer it from the account name?

Reply:
Ok. I'm just inferring it from the account name for now. I don't love that this knowledge (which accounts get the token) is leaking out...but it works. 

The regex is `^byu-oit-`. Once ACS is adding the token to CES accounts, we'll make it `^byu-oit-|^ces-`.